### PR TITLE
check paths-ignore for PR workflow

### PR DESCRIPTION
**Pull request by ndukalski**

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -8,14 +8,29 @@ on:
...
   # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:    
+  workflow_dispatch:
 
 permissions:
   contents: read
 
 jobs:
-  build:
+  check_changes:
```
Comment (karlisf) [inline .github/workflows/pr.yml:17]: check_workflow_changes

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -8,14 +8,29 @@ on:
...
+    outputs:
+      not_workflows: ${{ steps.filter.outputs.not_workflows }}
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            not_workflows:
```
Comment (karlisf) [inline .github/workflows/pr.yml:28]: naming suggestion: no_workflow_changes

File: .github/workflows/pr.yml (hunk 1 above)
Comment (ndukalski) [reply .github/workflows/pr.yml]: done

**Review by karlisf — APPROVED** [review]

File: .github/workflows/pr.yml (hunk 2 above)
Comment (ndukalski) [reply .github/workflows/pr.yml]: done

**Pull request merged**